code/controllers/subsystems/ticker.dm

```diff
@@ -450,6 +450,9 @@ Helpers
 			to_world(minions)
 
 	var/dronecount = 0
+	//tell the nice people on discord what went on before the salt cannon happens.
+	world.TgsTargetedChatBroadcast("The current round has ended. Please standby for the Foundation's report!", FALSE)
```

Comment:
Maybe we should have some round info here? e.g. gamemode, max player count, etc.